pkg/commands/git_commands/branch_loader.go

```diff
@@ -188,20 +189,23 @@ var branchFields = []string{
 }
 
 // Obtain branch information from parsed line output of getRawBranches()
-func obtainBranch(split []string) *models.Branch {
+func obtainBranch(split []string, storeCommitDateAsRecency bool) *models.Branch {
 	headMarker := split[0]
 	fullName := split[1]
 	upstreamName := split[2]
 	track := split[3]
 	subject := split[4]
 	commitHash := split[5]
-	commitDate := split[6]
```

Comment:
I'd prefer if this stays here; it's good to see how the fields are unpacked in one place. This requires adding a time stamp to the mock data of all tests, which is preferable too because it's closer to what git returns in production. (Changed in 5f272433fabd)

Reply:
Fair enough, if there is no problem adding the 2h timestamp to other scenarios in `branch_loader_test.go`.